docs/changelog/101904.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 101904
+summary: Grant api key api with jwt
```

Comment:
nit: (because these get put as-is to release notes) Allow granting API keys with JWT as the access_token

Reply:
Thanks for spotting and for the suggestion!